nltk/parse/dependencygraph.py

```diff
@@ -543,6 +543,109 @@ def nx_graph(self):
+        ...                               u'Lemma': u'.',
+        ...                               u'NamedEntityTag': u'O',
+        ...                               u'PartOfSpeech': u'.'}]]}
+        >>> list(stanfordjson2conll(stanfordjson)) # stanford-corenlp-full-2014-08-27.
+        [(2, '_', 'world', 'world', 'NN', 'NN', '_', '0', 'ROOT', '_', '_'), (1, '_', 'Hello', 'hello', 'UH', 'UH', '_', '2', 'discourse', '_', '_')]
+        >>> stanfordjson = {u'dependencies': [[u'root', u'ROOT', u'0', u'loves', u'2'],
+        ...                                    [u'nsubj',
+        ...                                     u'loves',
+        ...                                     u'2',
+        ...                                     u'John',
+        ...                                     u'1'],
+        ...                                    [u'dobj', u'loves', u'2', u'Mary', u'3'],
+        ...                                    [u'punct', u'loves', u'2', u'.', u'4']],
+        ...                  u'parsetree': [],
+        ...                  u'text': u'John loves Mary.',
+        ...                  u'words': [[u'John',
+        ...                              {u'CharacterOffsetBegin': u'0',
+        ...                               u'CharacterOffsetEnd': u'4',
+        ...                               u'Lemma': u'John',
+        ...                               u'PartOfSpeech': u'NNP'}],
+        ...                             [u'loves',
+        ...                              {u'CharacterOffsetBegin': u'5',
+        ...                               u'CharacterOffsetEnd': u'10',
+        ...                               u'Lemma': u'love',
+        ...                               u'PartOfSpeech': u'VBZ'}],
+        ...                             [u'Mary',
+        ...                              {u'CharacterOffsetBegin': u'11',
+        ...                               u'CharacterOffsetEnd': u'15',
+        ...                               u'Lemma': u'Mary',
+        ...                               u'PartOfSpeech': u'NNP'}],
+        ...                             [u'.',
+        ...                              {u'CharacterOffsetBegin': u'15',
+        ...                               u'CharacterOffsetEnd': u'16',
+        ...                               u'Lemma': u'.',
+        ...                               u'PartOfSpeech': u'.'}]]}
+        >>> list(stanfordjson2conll(stanfordjson)) # stanford-corenlp-full-2015-04-20.
+        [(2, '_', 'loves', 'love', 'VBZ', 'VBZ', '_', '0', 'ROOT', '_', '_'), (1, '_', 'John', 'John', 'NNP', 'NNP', '_', '2', 'nsubj', '_', '_'), (3, '_', 'Mary', 'Mary', 'NNP', 'NNP', '_', '2', 'dobj', '_', '_'), (4, '_', '.', '.', '.', '.', '_', '2', 'punct', '_', '_')]
+        
+        :param sentence: The JSON output from Stanford CoreNLP tool.
+        :type sentence: dict
+        :rtype: generator
+        """
+        for node in jsonparse['dependencies']:
```

Comment:
I believe that in recent corenlp version the JSON format is different, or the wrapper outputs something else.

Have a look to https://github.com/nltk/nltk/pull/1249/files#diff-3ddcbc41d8654bc0748b8afb087c55aeR633 it would be cool to have one function that does the job.